shared/message/platform/NUSenseData.proto

```diff
@@ -91,6 +91,19 @@ message IMU {
     uint32 temperature = 3;
 }
 
+message BackPanel {
+
+    // For now, I am only including the two buttons since I see no current need to poll the state of the LEDs, and I
+    // want to conserve as much bandwidth as possible. If such a need arises, then one can add it under the BackPanel
+    // message.
```

Comment:
probably just simplify this to "LEDs not added to conserve bandwidth, but can be added under the BackPanel if needed". Or attribute yourself to the "I"s.

Reply:
Done.